blog/2023-03-04-interact-with-celo-blockchain-using-web3dart/interact-with-celo-blockchain-using-web3dart.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Interact with Celo Blockchain using web3dart
+description: This article explains how to construct a dart program to communicate with the Celo blockchain using web3dart
+
```

Comment:
Hey Its still not compiling can we try something I don't know it should make issue or not but can you remove this blank line from header we might get solved issue because I am not being able to find anything else in this issue.